Cargo.toml

```diff
@@ -13,9 +13,22 @@ Simple and fast implementation of language tag normalization and validation
 """
 edition = "2018"
 
+[features]
+default = []
+serialize = ["serde/derive"]
```

Comment:
I believe we don't need Serde `derive` feature anymore because we only "manually" implement `Serialize` and `Deserialize`.
We might only keep serde as an optional dependency and enable `Serialize` and `Deserialize` if it is available.
What do you think about it?

Reply:
Oops, forgot about that :p. Thanks, will fix.